src/test/app.test.js

```diff
@@ -16,3 +16,16 @@ describe('TESTING ENDPOINTS', () => {
       });
   });
 });
+
+describe('TESTING LOGOUT', () => {
```

Comment:
we should move this test suite to its own file

Reply:
okay will do now...